app/uk/gov/hmrc/nisp/views/account.scala.html

```diff
@@ -56,140 +55,63 @@
-                            </li>
-                        }
-                    }
-                    <li>
-                        <a href="#how-to-increase-it" role="tab" id="tab-how-to-increase-it">@Messages("nisp.main.howtoincreaseit")</a>
-                    </li>
-                </ul>
-            </div>
-            @for(test <- abTest) {
-                @if(test == B && spSummary.contractedOutFlag) {
-                    <div class="tab-panel" id="cope" role="tabpanel">
-                        <div class="content-block">
-                            <h2 class="heading-medium">@Messages("nisp.cope.title1")</h2>
-                            <p>@Messages("nisp.cope.inthepast")</p>
-                            <p>@Messages("nisp.cope.why")</p>
-                            <ul class="list-bullet">
-                                <li>@Messages("nisp.cope.why.bullet1")</li>
-                                <li>@Messages("nisp.cope.why.bullet2")</li>
-                            </ul>
-                            <h2 class="heading-medium">@Messages("nisp.cope.title2")</h2>
-                            <p>@Messages("nisp.cope.definition")</p>
-                            <p>@Messages("nisp.cope.workplace")</p>
-
-                            <table class="cope-table">
-                                <thead>
-                                    <tr>
-                                        <th>&nbsp;</th>
-                                        <th class="cope-table-number">@Messages("nisp.main.chart.week")</th>
-                                    </tr>
-                                </thead>
-                                <tbody>
-                                <tr>
-                                    <td>@Messages("nisp.cope.table.pensionforecast")</td>
-                                    <td class="cope-table-number">@NispMoney.pounds(spSummary.forecastAmount.week)</td>
-                                </tr>
-                                <tr>
-                                    <td>@Messages("nisp.cope.table.estimate.title")</td>
-                                    <td class="cope-table-number">@NispMoney.pounds(spSummary.copeAmount.week)</td>
-                                </tr>
-                                <tr>
-                                    <td>@Messages("nisp.cope.table.copeadded")</td>
-                                    <td class="cope-table-number">@NispMoney.pounds(spSummary.copeAmount.week+spSummary.forecastAmount.week)</td>
-                                </tr>
-                                </tbody>
-                            </table>
-
-                            <p>@Html(Messages("nisp.main.cope.linktitle"))</p>
-
-                        </div>
-                    </div>
-               }
+              articleClasses = Some("mainpage"), analyticsAdditionalJs = Some(analyticsAdditionalJs)) {
+
+    <h1 class="heading-large">@Messages("nisp.main.h1.title")</h1>
```

Comment:
This can be added to the `nispMain` call as `pageTitle = Messages("nisp.main.h1.title")`.

If you look at `nispMain` you can see it outputs a `<h1 class="heading-large">`